starknet_py/net/full_node_client.py

```diff
@@ -454,3 +446,53 @@ def _get_call_payload(tx: Union[InvokeFunction, Call]) -> dict:
         "entry_point_selector": convert_to_felt(tx.selector),
         "calldata": [convert_to_felt(i) for i in tx.calldata],
     }
+
+
+def _create_broadcasted_txn(transaction: Union[InvokeFunction, Declare]) -> Dict:
+    common_params = {
+        "type": "INVOKE",
```

Comment:
Is `"type"` a common param? I don't think a `Declare` transaction version should be `INVOKE`.